src/ts/ConvivaAnalytics.ts

```diff
@@ -427,8 +449,6 @@ export class ConvivaAnalytics {
   private registerPlayerEvents(): void {
     let player = this.player;
     let playerEvents = this.playerEvents;
-    playerEvents.add(player.EVENT.ON_SOURCE_LOADED, this.onSourceLoaded);
```

Comment:
Is it really ok to remove these? Don't we have to track the time between loading the player and starting playback as well? What if we have a preroll ad, is the metadata still correctly set?

Reply:
No we don't need to track time between loading and player start playback. Only interesting value here is VST (Video Startup Time) Time between User initializes play and the player starts playback. Therefore it was suggested to track if preload / autoplay is enabled because this is effecting the VST the most.

Reply:
👍